krux/python-krux-boto
# No warn region

## What does this PR do?

Changes logging of a missing default region from `WARN` to `DEBUG`.

## Why is this change being made?

Reduced noise in system logs.

## How was this tested? How can the reviewer verify your testing?

* build monitoring-scripts with the working copy of this branch
* verify there is no `There is not a default region set in your environment variables.` message in /var/log/syslog

## What gif best describes this PR or how it makes you feel?

<img src="https://media1.giphy.com/media/Xtqfb8XWNEpX2/giphy.gif" />

## Completion checklist

- [x] The pull request has been appropriately labeled according to
  our [conventions](https://kruxdigital.jira.com/wiki/display/EN/Changes+and+Peer+Review).
- [x] The version of the package has been updated according to [Semantic Versioning](http://semver.org/).
- [x] The change has unit & integration tests as appropriate.
- [x] Documentation is up to date and correct.
- [x] Dependencies are correctly listed under `requirements.pip` and
      are up to date without going over a major version.
- [x] Stakeholders have been notified. Workflow-impacting changes have
      been appropriately socialized to avoid surprises.
- [x] The change is either small or have been canaried in the appropriate environment(s).
